backend/api/gitops/webhook.go

```diff
@@ -1600,39 +1600,58 @@ func (s *Service) createIssueFromMigrationDetailsV2(ctx context.Context, project
 			},
 		}
 	} else {
-		var specs []*v1pb.Plan_Spec
+		environments, err := s.store.ListEnvironmentV2(ctx, &store.FindEnvironmentMessage{})
+		if err != nil {
+			return err
+		}
+		sort.Slice(environments, func(i, j int) bool {
+			return environments[i].Order < environments[j].Order
+		})
```

Comment:
Looks like we don't need the sort here? https://sourcegraph.com/github.com/bytebase/bytebase/-/blob/backend/store/environment.go?L325